protocol/testutil/keeper/prices.go

```diff
@@ -134,20 +210,16 @@ func CreateNMarkets(t testing.TB, ctx sdk.Context, keeper *keeper.Keeper, n int)
 	for i := range items {
 		items[i].Param.Id = uint32(i) + numExistingMarkets
 		items[i].Param.Pair = fmt.Sprintf("%v-%v", i, i)
-		items[i].Param.Exponent = int32(i)
+		items[i].Param.Exponent = -int32(i%36 + 1) // must be between -1 and -36
```

Comment:
There is validation on the marketmap side that `Decimals` (equivalent to Exponent * -1) is between 1 and 36